examples/hello-world/step-by-step/cifar10/sag_executor/sag_executor.ipynb

```diff
@@ -222,7 +222,12 @@
    "id": "48271064",
    "metadata": {},
    "source": [
-    "For additional resources, take a look at the various other executors with different use cases in the app_common, app_opt, and examples folder."
+    "For additional resources, take a look at the various other executors with different use cases in the app_common, app_opt, and examples folder.\n",
+    "\n",
+    "In the previous examples we have finished covering each of Execution API types: the Client API, Model Learner, and Executor.\n",
+    "Now we will revert back to using the Client API in future examples to highlight other features and workflows.\n",
```

Comment:
"revert back" or "stick to"? revert sounds like we are backing, not progressing, this is subjective though

Reply:
I agree, changed wording